dashboard/app/models/pd/application/facilitator1819_application.rb

```diff
@@ -38,6 +38,16 @@ module Pd::Application
   class Facilitator1819Application < ApplicationBase
     include Facilitator1819ApplicationConstants
 
+    def send_decision_notification_email
+      # Accepted, declined, and waitlisted are the only valid "final" states;
+      # all other states shouldn't need emails, and we plan to send "Accepted"
+      # emails manually
+      return unless %w(declined waitlisted).include?(status)
```

Comment:
should we have something like a `final_status?` check?

Reply:
We might possibly want one for another purpose, but it's worth noting that "accepted" is also a valid final status, just one for which we don't want to send emails automatically. I think I'd prefer to keep this check in here as-is, since it is specifically a "is the status one for which we want to send an email" check